### PR TITLE
E2E for DCGAN + adding zero filled bias node for the ConvTranspose when bias=False

### PR DESCRIPTION
**Pull request by prigoyal**
C2 requires 3 inputs for ConvTranspose even though bias=False. For now, we just create dummy 0 bias node and tested DCGAN models e2e fine

### CODE REVIEW

File: torch/csrc/autograd/functions/toffee/convolution.cpp [hunk 1]
```diff
@@ -30,6 +30,16 @@ jit::node_list ConvForward::primspec(PrimSpecContext* ctx, jit::node_list inputs
...
 
+  // For ConvTranspose, we append zero filled bias if the bias=False to maintain
+  // compatibility with Caffe2
+  if(transposed) {
+    if(inputs.at(2)->kind() == jit::kUndefined) {
+      n->addInput(g->appendNode(g->createConstant(at::CPU(at::kFloat).zeros({weight_size[1]}))));
+    }
+  }
+
+  g->appendNode(n);
```
Comment (ezyang) [inline torch/csrc/autograd/functions/toffee/convolution.cpp:41]: I see why you ran into a lint failure :)

File: torch/csrc/jit/ir.cpp [hunk 2]
```diff
@@ -353,6 +353,7 @@ void Node::lint() {
 void Graph::lint() {
   // Graph invariants
 
+  // Incomment the following to see the graph
```
Comment (ezyang) [inline torch/csrc/jit/ir.cpp:356]: Someday we will actually catch the assert errors and print the graph out automatically.

File: test/model_defs/caffe2_pytorch_test_models.py [hunk 3]
```diff
@@ -19,16 +22,10 @@
...
 
 skip = unittest.skip
 
-try:
-    import caffe2
-except ImportError:
-    print('Cannot import caffe2, hence caffe2-torch test will not run.')
-    sys.exit(0)
-
+torch.set_default_tensor_type('torch.FloatTensor')
```
Comment (fmassa) [inline test/model_defs/caffe2_pytorch_test_models.py:28]: Isn't `FloatTensor` the default tensor type in pytorch?

Comment (ezyang) [reply test/model_defs/caffe2_pytorch_test_models.py:28]: In the test suite (test/common.py), we set it to be DoubleTensor to improve precision. This is not good when you want to run pretrained models :) OTOH, I don't think this file is importing common.py

**Comment by ezyang** [comment]
Merged.

**Pull request closed without merging**